config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -0,0 +1,236 @@
+#! /bin/bash -e
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The presubmit jobs for the different Kubernetes-CSI repos are all
+# the same except for the repo name. As Prow has no way of specifying
+# the same job for multiple repos and manually copy-and-paste would be
+# tedious, this script is used instead to generate them.
+
+base="$(dirname $0)"
+
+# All kubernetes-csi repos for which Prow gets configured.
+repos="
+cluster-driver-registrar
+csi-driver-fibre-channel
+csi-driver-host-path
+csi-driver-image-populator
+csi-driver-iscsi
+csi-driver-nfs
+csi-lib-common
+csi-lib-fc
+csi-lib-iscsi
+csi-lib-utils
+csi-release-tools
+csi-test
+external-attacher
+external-provisioner
+external-resizer
+external-snapshotter
+kubernetes-csi-migration-library
+livenessprobe
+node-driver-registrar
+"
+
+# All branches that do *not* support Prow testing. All new branches
+# are expected to have that support, therefore these list should be
+# fixed. By blacklisting old branches we can avoid Prow config
+# changes each time a new branch gets created.
+skip_branches_cluster_driver_registrar='^(release-1.0)$'
+skip_branches_csi_lib_utils='^(release-0.1|release-0.2)$'
+skip_branches_csi_test='^(release-0.3|release-1.0|saad-ali-patch-1|saad-ali-patch-2|v0.1.0|v0.2.0)$'
```

Comment:
I will take an action item to delete these non-release branches

Reply:
Good plan :+1:  I just ran a short script that collected all existing branches, without manually pruning.